src/lib/utils/evm.ts

```diff
@@ -226,7 +226,11 @@ export const getApproveContractData = ({
   spender,
 }: GetApproveContractDataArgs): string => {
   const address = ethers.utils.getAddress(to)
-  const contract = getOrCreateContractByType({ address, type: ContractType.ERC20 })
+  const contract = getOrCreateContractByType({
+    address,
+    type: ContractType.ERC20,
+    chainId: KnownChainIds.EthereumMainnet,
```

Comment:
Also needs to be programmatic - can get `chainId` from `sellAsset`: https://github.com/shapeshift/web/blob/97162078f215aff54da0a3bbc0b9dabec1e9c68f/src/components/MultiHopTrade/hooks/useAllowanceApproval/helpers.ts#L36